src/web/upload/upload_bot_new.py

```
@@ -0,0 +1,66 @@
+            response = site.upload(
+                # file=(os.path.basename(file_path), file_content, 'image/svg+xml'),
+                file=f,
+                filename=file_name,
+                comment=summary or "",
+                ignore=True  # skip warnings like "file exists"
+            )
+
+        logger.debug(f"Successfully uploaded {file_name} to Wikimedia Commons")
+        return response
+    except requests.exceptions.HTTPError:
+        logger.error("HTTP error occurred while uploading file")
```

Ruff (0.14.1)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)